cms/bundles/viewsets/bundle.py

```diff
@@ -206,17 +207,6 @@ def get_field_label(self, field_name: str, field: "Field") -> str:
             case _:
                 return super().get_field_label(field_name, field)  # type: ignore[no-any-return]
 
-    def get_field_display_value(self, field_name: str, field: "Field") -> Any:
```

Comment:
Since this has been removed, it would be good to explain the magical name resolution that's happening in the background thanks to Wagtail's [get_field_display_value](https://github.com/wagtail/wagtail/blob/a783b8c3e713b119c4f72d037d3b9489b80854e3/wagtail/admin/views/generic/models.py#L1105), perhaps even include a link. 
I think the appropriate place for it could be the `get_fields` function, with the docstring something along the lines of:
```
        """
        Returns the list of fields to include in the inspect view.
        Values are inserted by methods following the get_FIELDNAME_display_value pattern.
        """
```